src/generator/time.ts

```diff
@@ -21,22 +22,19 @@ export class Content {
 // Creates the content for required time marshalling helpers.
 // Will be empty if no helpers are required.
 export async function generateTimeHelpers(session: Session<CodeModel>): Promise<Content[]> {
-  const headerText = comment(await session.getValue("header-text", "MISSING LICENSE HEADER"), "// ");
-  const namespace = await session.getValue('namespace');
+  const preamble = await contentPreamble(session);
   const content = new Array<Content>();
   if (session.model.language.go!.hasTimeRFC1123) {
-    content.push(new Content('time_rfc1123', generateRFC1123Helper(headerText, <string>namespace)));
+    content.push(new Content('time_rfc1123', generateRFC1123Helper(preamble)));
   }
   if (session.model.language.go!.hasTimeRFC3339) {
-    content.push(new Content('time_rfc3339', generateRFC3339Helper(headerText, <string>namespace)));
+    content.push(new Content('time_rfc3339', generateRFC3339Helper(preamble)));
   }
   return content;
 }
 
-function generateRFC1123Helper(header: string, packageName: string): string {
-  return `${header}
-
-package ${packageName}
+function generateRFC1123Helper(preamble: string): string {
+  return `${preamble}
```

Comment:
Semi colon

Reply:
It's at the very end of the text (same with other type).

Reply:
Oops you're right I see now😅